app/src/test/java/org/fossasia/openevent/app/presenter/AttendeePresenterTest.java

```diff
@@ -82,20 +82,6 @@ public void shouldLoadAttendeesAutomatically() {
     }
 
     @Test
-    public void shouldDetachViewOnStop() {
```

Comment:
Never ever delete tests before completely being sure that it isn't needed. We need to always detach presenters at correct lifecycle method. Even for the MainActivity whose life span is complete lifespan of the application. Still need to detach it